dotCMS/src/main/java/com/dotcms/contenttype/business/uniquefields/extratable/UniqueFieldDataBaseUtil.java

```diff
@@ -0,0 +1,73 @@
+package com.dotcms.contenttype.business.uniquefields.extratable;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static com.dotcms.util.CollectionsUtils.list;
+
+/**
+ * Util class to handle QL statement related with the unique_fiedls table
+ */
+enum UniqueFieldDataBaseUtil {
```

Comment:
this does not looks such as enum, may be a class and use the CDI

Reply:
https://dzone.com/articles/java-singletons-using-enum

Reply:
-  done https://github.com/dotCMS/core/pull/30466/files#diff-293616ea70541ed645d776bdcefeba97c7a94b780f5aaa63487729fdd6d00a9dR20